child_compassion/demo/compassion.child.csv

```diff
@@ -0,0 +1,67 @@
+id,code,name
```

Comment:
At first, it is what I have done, but it seems that they are not updated (or badly)